src/librawspeed/decoders/DngDecoder.cpp

```diff
@@ -206,8 +206,10 @@ void DngDecoder::parseCFA(const TiffIFD* raw) const {
       }))
     ThrowRDE("Error decoding active area");
 
-  mRaw->cfa.shiftRight(aa[1]);
-  mRaw->cfa.shiftDown(aa[0]);
+  // To reverse the ActiveArea modifictions done earlier, we need to
```

Comment:
Apologies for continuing to bikesheed, but i still don't quite get the whole picture.
Is the `TiffTag::CFAPATTERN` in DNG specified for `(0,0)`, or for `TiffTag::ACTIVEAREA`?

Additionally, `RawImageData::subFrame()` says:
https://github.com/darktable-org/rawspeed/blob/586d1492492d9df8a7683401f3017c54a770ffce/src/librawspeed/common/RawImage.cpp#L261-L265

Reply:
CFAPATTERN is specified for ACTIVEAREA.